packages/contracts/test/diamond/DiamondTestSetup.sol

```diff
@@ -387,6 +399,14 @@ abstract contract DiamondTestSetup is DiamondTestHelper, UUPSTestHelper {
             })
         );
 
+        cuts[19] = (
```

Comment:
You may keep the `PoolLiquidityMonitor` contract out of the diamond (i.e. as a standalone contract, not as a facet). I think it's simpler to maintain it as a standalone contract.